ogr/services/gitlab/pull_request.py

```diff
@@ -103,6 +110,19 @@ def commits_url(self) -> str:
     def head_commit(self) -> str:
         return self._raw_pr.sha
 
+    @property
+    def merge_commit_sha(self) -> str:
+        return self._raw_pr.merge_commit_sha
+
+    @property
+    def merge_commit_status(self) -> MergeCommitStatus:
+        status = self._raw_pr.merge_status
+        assert status in self._merge_commit_status
```

Comment:
```suggestion
```

Using `assert` in the production code is not expected -- python, when run in the optimise mode, ignores such commands so we can't really on the execution of this.

Note that you are checking this one line bellow so it's not needed here.

Reply:
Fixed by myself.